code/_onclick/cyborg.dm

```diff
@@ -83,6 +83,15 @@
 		if(CanReach(A,W))
 			W.melee_attack_chain(src, A, params)
 			return
+		else
+			if(LAZYACCESS(modifiers, RIGHT_CLICK))
+				var/after_attack_secondary_result = W.afterattack_secondary(A, src, FALSE, params)
+
+				if(after_attack_secondary_result == SECONDARY_ATTACK_CALL_NORMAL)
+					W.afterattack(A, src, FALSE, params)
+			else
+				W.afterattack(A, src, FALSE, params)
+
 		if(isturf(A) || isturf(A.loc))
 			W.afterattack(A, src, 0, params)
```

Comment:
This line looks suspicious now

Reply:
I updated the check but honestly it's probably not needed anymore? _I barely know what I'm doing_

Reply:
Well it's prevent borgs from afterattacking atoms in their contents, so who knows